### ISSUE TITLE
How do I install ppx_show on my machine?

### ISSUE THREAD

**Issue by xennygrimmato** [diml/ppx_show#1]
Can you provide installation instructions for ppx_show?
I ran 'make' in the repo, and got the following output:
```bash
File "src/gen_printer.ml", line 106, characters 28-35:
Error: Unbound constructor Nolabel
Command exited with code 2.
Compilation unsuccessful after building 8 targets (0 cached) in 00:00:00.
make: *** [all] Error 10
```

**Comment by xennygrimmato** [comment]
I realized that I was using ocaml `4.02.3`. After switching to `4.03.0`, I got the following error on performing `make`:

```ocaml
File "src/gen_printer.ml", line 96, characters 33-45:
Error: This variant expression is expected to have type Parsetree.constant
       The constructor Const_string does not belong to type Parsetree.constant
Hint: Did you mean Pconst_string?
```

**Comment by diml** [comment]
Indeed, this was written for a non-released version of 4.03. I fxed it in bf2ff1a24572db1f647fdd8e0fef44f392ed7b0d. Note that I'm not actively maintaining this package, but if you are interested you are welcome to fork it and release it

**Comment by xennygrimmato** [comment]
Thanks for the fix!
